src/frontend/components/Categories/Categories.jsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import "./Categories.css";
+
+const Categories = () => {
+  return (
+    <div className="featured-categories flex-dir-col">
+      <h1>Featured Categories</h1>
+      <p className="text-medium">
+        Below are the trending featured products of Local Tailor
+      </p>
+      <div className="categories flex-dir-row">
+        <p className="categorie">
+          <a className="text-decoration-none" href="/products">
+            Men
+          </a>
+        </p>
+        <p className="categorie">
```

Comment:
You can avoid code repetition here, you can add category at categories.js Or products.js in db folder & then map them over the HTML code